livekit-agents/livekit/agents/cli/cli.py

```diff
@@ -1205,7 +1205,7 @@ async def _async_main() -> None:
                     self._shutdown_cb()
                     return
 
-            self._server._job_executor_type = JobExecutorType.THREAD  # TODO: better setter
+            self._server.job_executor_type = JobExecutorType.THREAD
```

Comment:
@theomonnom Is it that in this case this should be `self._server.update_options()`?